### ISSUE TITLE
[Android Bootcamp] Project 0: Todo - Ready to review

### ISSUE THREAD

**Issue by boli2659** [boli2659/codepathSimpleToDo#1]
My app is complete- please review. /cc @codepathreview @codepath

One question I had about this was the OptionsMenu method/objects, because I did not use them. In the slides, there was screenshot at the initial wizard setup of the project where it had a place to enter menu options when it asked for the name of the Main Activity, but that must be in an older version of Android Studio, because I did not have that option.

**Comment by nesquena** [comment]
Hi Zach,

Can you update your project by copying over and filling out this [README template](http://courses.codepath.com/snippets/intro_to_android/readme_templates/prework_readme.md?raw=true) and recording the GIF walkthrough as [specified here](http://courses.codepath.com/snippets/intro_to_android/prework#heading-4-submitting-the-project). Ping me back here once that's updated. Thanks!

**Comment by nesquena** [comment]
> One question I had about this was the OptionsMenu method/objects, because I did not use them. In the slides, there was screenshot at the initial wizard setup of the project where it had a place to enter menu options when it asked for the name of the Main Activity, but that must be in an older version of Android Studio, because I did not have that option.

Yep that's right, it was changed in a recent version. We'll learn more about them in the class, you only need the menu methods when [configuring the top action bar](http://guides.codepath.com/android/Defining-The-ActionBar)

**Comment by boli2659** [comment]
All right awesome, taken care of. Thanks!

**Comment by nesquena** [comment]
Zach,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer  learn the practical elements of Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo) and to **schedule a short 15 minute phone conversation** with us. Navigate to [today's date and onward from there](https://calendar.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU) and choose an available 15-minute slot in the mornings or evenings. Let us know if none of those times work.

With regards to extensions to implement, we are looking for developers with an interest in user experience and product development so be sure to focus on additional functional enhancements and/or improving the user interface of the app.

Once you select a slot, can you make sure to **include the best number** to reach you at to the invite?

**Prior to our chat**, can you take some time to research the most common Android Activity lifecycle methods as well as the `ArrayAdapter` pattern used when working with lists? We will be discussing these topics with you conceptually in the phone call. We encourage you to [search our guides](http://guides.codepath.com/android/) as well as any other sources you'd like.

Look forward to chatting soon!

**Comment by nesquena** [comment]
P.S I still can't see your recorded GIF for the app in the README. Can you replace the fake URL in the README with your actual app GIF before our call. Thanks!

**Comment by boli2659** [comment]
Sure thing. Must've forgot to git add the file, sorry about that. But everything is taken care of! Looking forward to talking as well!

**Comment by nesquena** [comment]
Great, thanks. Chat with you soon.

**Comment by nesquena** [comment]
@boli2659 can you email me with the best number to reach you at for our call?

**Comment by boli2659** [comment]
I added functionality for most of the optional aspects, and I will be adding the dialog fragment tomorrow. Looking forward to class Monday!

**Comment by nesquena** [comment]
Great, thanks for the update. Hope you were able to gain a deeper understanding of some of the concepts by extending the pre-work. See you tomorrow!
